.github/workflows/molecule.yml

```diff
@@ -13,10 +13,10 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
-      max-parallel: 4
+      max-parallel: 1
       matrix:
         distro: [rockylinux8]
-        scenario: [default]
+        scenario: [default, renew, ca-renew]
```

Comment:
I love that you added test scenarios! Just wanted to say that, you can close the conversation right after reading it. :-)